docs/run-a-node/en/how-to-get.md

```diff
@@ -0,0 +1,55 @@
+---
+id: how_to_get
+title: Get Conflux Client
+custom_edit_url: https://github.com/Conflux-Chain/conflux-developer-site/edit/master/docs/run-a-node/en/how-to-get.md
+keywords:
+  - node
+---
+
+[Conflux-rust](https://github.com/conflux-chain/conflux-rust) is a Conflux protocol high-performance implementation developed by the foundation with the Rust language. It is an Conflux Client program that runs Conflux network nodes and provides RPC service to the public. It is also a command-line tool that manages local accounts and conducts PRC interaction with nodes. You can download the release package through any of the ways listed: 
```

Comment:
> is a Conflux protocol high-performance implementation

is a high-performance implementation of the Conflux protocol

> It is an Conflux Client program that runs Conflux network nodes and provides RPC service to the public.

Conflux-rust is a Conflux node that connects to other nodes on the Conflux network and provides an RPC interface that you can use to query and interact with the blockchain.

> release package

released package